tests/adapter/mongo/test_query.py

```diff
@@ -1,9 +1,12 @@
+import logging
 import re
 
 from pymongo import ReturnDocument
 
 from scout.constants import CLINSIG_MAP, TRUSTED_REVSTAT_LEVEL
 
+LOG = logging.getLogger(__name__)
+
```

Comment:
```suggestion
LOG = logging.getLogger(__name__)

```

Reply:
You mean to remove it, right? I agree, it's not really used any more. 😊